corehq/apps/app_manager/views/apps.py

```diff
@@ -254,7 +254,10 @@ def _get_setting(setting_type, setting_id):
         )
     })
     context.update({
-        'smart_lang_display_enabled': app.smart_lang_display
+        'smart_lang_display_enabled': app.smart_lang_display,
+        'show_ui_notification_to_hide_translations': (
```

Comment:
For some reason I had in my head that getting rid of `custom_checks` would allow logic to be only in the template without anything in the context...forgot that django templates can't handle `>`. In any case, I do think it's simpler this way, thank you for updating.